ITensorGaussianMPS/src/gmps.jl

```diff
@@ -135,6 +135,15 @@ is_annihilation_operator(::OpName"c") = true
 is_annihilation_operator(::OpName"c↑") = true
 is_annihilation_operator(::OpName"c↓") = true
 
+expand_to_ladder_operators(o::Op) = expand_to_ladder_operators(ITensors.name(o))
+expand_to_ladder_operators(o::String) = expand_to_ladder_operators(OpName(o))
+expand_to_ladder_operators(opname::OpName) = opname # By default does nothing
+expand_to_ladder_operators(::OpName"N") = ["Cdag", "C"]
+expand_to_ladder_operators(::OpName"Nup") = ["Cdagup", "Cup"]
+expand_to_ladder_operators(::OpName"n↑") = ["Cdagup", "Cup"]
```

Comment:
Should be able touse:
```julia
expand_to_ladder_operators(opname::OpName"n↑") = expand_to_ladder_operators(alias(opname))
```

Reply:
seems `alias` doesn't get exported so hopefully you don't mind the import (otherwise JuliaFormatter makes this into a multiline function)

Reply:
Let's use `using ITensors: alias` instead of `import ITensors: alias`. I try to only `import` if I'm actually overloading a function.